purchase_line_procurement_group/i18n/purchase_line_procurement_group.pot

```diff
@@ -0,0 +1,35 @@
+# Translation of Odoo Server.
```

Comment:
remove the file

Reply:
@i-vyshnevska: this file defines the template for the rest .po file. Why should we remove this file ? I see most modules have this file added. Thanks